capnpc-go/templates/constants

```diff
@@ -12,3 +12,11 @@ var (
 {{end}}
 )
 {{end}}
+
+{{with .Vars -}}
+func init() {
+   // Set traversal limit for constants as Uint64Max since they're safe from amplification attacks
+  {{range .}} {{.Name}}.Segment().Message().ReadLimiter().Reset((1<<64) - 1)
```

Comment:
I'd like to avoid the generated blank line. Try this:

```
	// Set traversal limit for constants as Uint64Max since they're safe from amplification attacks.
	{{- range .}}
	{{.Name}}.Segment().Message().ReadLimiter().Reset((1<<64) - 1)
	{{- end}}
}
```